src/styles/blocks/card.scss

```
@@ -0,0 +1,121 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+
+  padding: 32px 16px 16px;
+  flex-wrap: wrap;
+  align-content: flex-start;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  color: #060b35;
+
+  transition: all 300ms ease-in-out;
+
+  &--width {
+    width: 200px;
+  }
+
+  &__image-container {
+    display: flex;
+    width: 100%;
+    justify-content: center;
+  }
+
+  &__image {
+    margin: 0 auto;
+    display: block;
+    height: 134px;
+    width: 160px;
+  }
+
+  &__name {
+    margin-top: 40px;
+    margin-bottom: 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin-top: 4px;
+    margin-bottom: 0;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__stars {
+    width: 100%;
+    height: 16px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0;
+    margin-top: 16px;
+  }
+
+  &__price {
+    width: 100%;
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__label {
+    margin-bottom: 0;
+    font-size: 12px;
+    font-weight: 400;
+    color: #616070;
+  }
+
+  &__amount {
+    margin-bottom: 0;
+    font-size: 16px;
+    font-weight: 700;
+  }
+
+  &__buy-button {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 16px;
+    width: 166px;
+    height: 40px;
+
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    border-radius: 5px;
+    color: #fff;
+    background: #00acdc;
+    text-decoration: none;
+
+    border: 1px solid #f3f3f3;
+
+    &:hover {
+      color: #00acdc;
+      border: 1px solid #00acdc;
```

```suggestion
      border-color: #00acdc;
```
There is no need to add a new border on hover, just change its color.
Also, make sure that all hover effects work smoothly